### PR TITLE
Ability to toggle secondary chart

### LINKED ISSUE #98: Hide secondary chart by default
By default, the secondary chart should be hidden. The user should have the option to turn RSI on/off, which will show/hide the secondary chart.

### PR DESCRIPTION
**Pull request by djmiley**
Resolves #98

### CODE REVIEW

**Comment by djmiley** [comment]
oops copuple of things to do here.

**Comment by djmiley** [comment]
Blocked by #125

File: src/assets/js/util/calculateDimensions.js [hunk 1]
```diff
@@ -13,18 +13,16 @@
...
-            .attr('height', mainHeightRatio * targetHeight / totalHeightRatio);
-        container.select('svg.rsi').attr('width', targetWidth)
-            .attr('height', rsiHeightRatio * targetHeight / totalHeightRatio);
-        container.select('svg.nav').attr('width', targetWidth);
+        container.select('svg.primary').attr('width', useableScreenWidth)
+            .attr('height', primaryHeightRatio * useableScreenHeight / totalHeightRatio);
+        container.select('svg.secondary').attr('width', useableScreenWidth)
+            .attr('height', secondaryHeightRatio * useableScreenHeight / totalHeightRatio);
+        container.select('svg.nav').attr('width', useableScreenWidth);
     };
```
Comment (djmiley) [inline src/assets/js/util/calculateDimensions.js]: Just renaming of variables, except for truthy on `secondaryChartShown`

File: src/assets/js/main.js [hunk 2]
```diff
@@ -92,18 +112,22 @@
...
 
         svgNav.datum(dataModel)
             .call(navChart);
     }
 
     function resize() {
-        sc.util.calculateDimensions(container);
+        sc.util.calculateDimensions(container, secondaryChart);
         render();
     }
```
Comment (djmiley) [inline src/assets/js/main.js]: Not certain about my treatment of initialisation and `null` values here.

Comment (janakerman) [inline src/assets/js/main.js]: I'm not actually sure what the best practises are with initialising things with no value. I normally simply do:

    var myVar;

I think it's fine to set it to null too. I suppose a bit of googling would answer that question, but I'm happy with what's here for the PR.

What do you mean about you 'treatment of initialisation'?

**Pull request merged**